app/cells/date_range/date_range.coffee

```diff
@@ -1,3 +1,9 @@
+#= require jquery
```

Comment:
I'll remove them for now.

It seems there's a code migration in progress [here](https://github.com/ad2games/ui_components/blob/115e6840ad043b75315e88c8dfbf0e696c00fdcc/spec/dummy/app/controllers/components_controller.rb#L2-L11), so new tests use a layout which only loads the component's js code(without jquery etc). Thus the need to require the dependencies(e.g. https://github.com/ad2games/ui_components/blob/115e6840ad043b75315e88c8dfbf0e696c00fdcc/app/cells/select/select.coffee)
But my attempt to go the "new way" failed because of the html-escaping bug in the slim.